src/config.cc

```diff
@@ -176,11 +176,36 @@ void Config::updateFromPropertyTree(const boost::property_tree::ptree& pt) {
   CopyFromConfig(rvi.packages_dir, "rvi.packages_dir", LVL_trace, pt);
   CopyFromConfig(rvi.uuid, "rvi.uuid", LVL_warning, pt);
 
+  CopyFromConfig(p11.module, "p11.module", LVL_trace, pt);
+  CopyFromConfig(p11.pass, "p11.pass", LVL_trace, pt);
+
   CopyFromConfig(tls.certificates_directory, "tls.certificates_directory", LVL_trace, pt);
   CopyFromConfig(tls.server, "tls.server", LVL_warning, pt);
-  CopyFromConfig(tls.ca_file, "tls.ca_file", LVL_warning, pt);
-  CopyFromConfig(tls.pkey_file, "tls.pkey_file", LVL_warning, pt);
-  CopyFromConfig(tls.client_certificate, "tls.client_certificate", LVL_warning, pt);
+
+  std::string tls_source = "file";
+  CopyFromConfig(tls_source, "tls.ca_source", LVL_warning, pt);
+  if (tls_source == "pkcs11")
+    tls.ca_source = kPkcs11;
+  else
+    tls.ca_source = kFile;
+
+  tls_source = "file";
+  CopyFromConfig(tls_source, "tls.cert_source", LVL_warning, pt);
+  if (tls_source == "pkcs11")
+    tls.cert_source = kPkcs11;
+  else
+    tls.cert_source = kFile;
+
+  tls_source = "file";
+  CopyFromConfig(tls_source, "tls.pkey_source", LVL_warning, pt);
+  if (tls_source == "pkcs11")
+    tls.pkey_source = kPkcs11;
+  else
+    tls.pkey_source = kFile;
+
+  CopyFromConfig(tls.ca_path, "tls.ca_path", LVL_warning, pt);
+  CopyFromConfig(tls.pkey_path, "tls.pkey_path", LVL_warning, pt);
+  CopyFromConfig(tls.cert_path, "tls.cert_path", LVL_warning, pt);
```

Comment:
Wouldn't it be better to avoid changing these configuration file names?  The Aktualizr recipe uses them today, and otherwise we will need to offer some backwards compatibility.

Reply:
But our sota.toml is in /usr now, we can safely update it together with the aktualizr binary.

Reply:
Yes, but we'll need to add both to meta-updater now, to support people who build latest master of Aktualizr.  What was the reason for changing the names?

Reply:
Ah, we have this bleeding edge build. OK, we can revert the names. I just thought that "path" sounds more generic than "file", which is good once PKCS#11 URI are supported. And this "client_certificate" field name was an annoying inconsistency as well.

Reply:
Yes, the name changes are nice, but last time with `certificates_path` I seemed to spend ages running into new places where the config file needed changing :)